javascript/javascript3/week3/homework/promise/main.js

```diff
@@ -0,0 +1,20 @@
+const div=document.createElement('div')
+async function fecthData(){
+  const response1=await  fetch('https://api.github.com/search/repositories?q=user:Vahablotfi')
+  const response2=await  fetch('https://api.github.com/search/repositories?q=user:mariem460')
+  const response3=await  fetch('https://api.github.com/search/repositories?q=user:asmafassy')
+
+  const user1=await response1.json()

```

Comment:
instead of making extra variable, use then like
```
fetch('https://api.github.com/search/repositories?q=user:Vahablotfi').then(response => response.json())
  ```